bot/admin/admin.py

```diff
@@ -0,0 +1,260 @@
+"""Contains a Cog for all administrative funcionality."""
+
+import json
+import typing
+from typing import Optional
+
+import discord
+import requests
+from discord.ext import commands
+
+from bot.logger import command_log
+
+
+class AdminCog(commands.Cog):
+    """Cog for administrative Functions."""
+
+    def __init__(self, bot):
+        """Initializes the Cog.
+
+        Args:
+            bot (discord.ext.commands.Bot): The bot for which this cog should be enabled.
+        """
+        self.bot = bot
+
+    # A special method that registers as a commands.check() for every command and subcommand in this cog.
+    async def cog_check(self, ctx):
+        return ctx.author == ctx.guild.owner  # Only the server owner can use the commands defined in this Cog.
```

Comment:
You do know, I can't develop anything in this cog or fix/change anything as long as this check exists?

Reply:
Yeah, I know that. But there won't be a lot of commands in there and the ones which are still missing will be coded by me anyway. But I'm gonna make #53 and #55 available to you, so that you can update the bot.